features/booking_admin.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Admin booking page
+  As an administrator
+  In order to sett up roles for booking a facilities
+  I would like to be able to manage the facilities booking settings.
+
+  Scenario: Create booking schedule for a facility
+    Given there is a facility named "Laundry Room"
+    And I am on the facility show page
+    And I click link "Laundry Room"
+    Then I should see "scheduleing controll for Laundry Room"
+    And I select "9" from "start_time"
+    And I select "5" from "time_slot"
+    And I select "3" from "hours"
+    And I click "Set schedule"
+    Then I should see "schedule created"
+    And I am on the facility show page
+    And I click link "Laundry Room"
+    Then I should not see "schedule created"
```

Comment:
So here you're checking that you don't see the flash message? It's unnecessary, really. Is there something else you can look for to see that the time slots have been created?